docs/en/sql-reference/functions/type-conversion-functions.md

```diff
@@ -4866,51 +4866,411 @@ Result:
 └───────┴───────────────┴──────┴──────────────┴──────────────┴──────────────────────┘
 ```
 
-## toInterval(Year\|Quarter\|Month\|Week\|Day\|Hour\|Minute\|Second)
+## toIntervalYear
 
-Converts a Number type argument to an [Interval](../data-types/special-data-types/interval.md) data type.
+Returns an interval of `n` years of data type [IntervalYear](../data-types/special-data-types/interval.md).
 
 **Syntax**
 
 ``` sql
-toIntervalSecond(number)
-toIntervalMinute(number)
-toIntervalHour(number)
-toIntervalDay(number)
-toIntervalWeek(number)
-toIntervalMonth(number)
-toIntervalQuarter(number)
-toIntervalYear(number)
+toIntervalYear(n)
 ```
 
 **Arguments**
 
-- `number` — Duration of interval. Positive integer number.
+- `n` — Number of years. Positive integer number or string representation thereof. [Int*](../data-types/int-uint.md)/[String](../data-types/string.md).
```

Comment:
@Blargian Could we re-phrase to

```md
Integer or float number, or string representation thereof. [(U)Int*](../data-types/int-uint.md)/[Float*](../data-types/float.md)/[String](../data-types/string.md).
```

? (asking because Floats and negative integers, even negative floats, work)

Reply:
@rschu1ze maybe `Integer numbers or string representations thereof, and float numbers`? I get an exception for string representations of floats: `SELECT toIntervalMonth('1.0');`

Reply:
@Blargian You are right, let's do as you propose.